homeassistant/components/tuya/sensor.py

```diff
@@ -510,25 +510,53 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
     # Smart Camera
     # https://developer.tuya.com/en/docs/iot/categorysp?id=Kaiuz35leyo12
     "sp": (
+        *BATTERY_SENSORS,
         TuyaSensorEntityDescription(
             key=DPCode.SENSOR_TEMPERATURE,
             name="Temperature",
+            icon="mdi:thermometer",
             device_class=SensorDeviceClass.TEMPERATURE,
             state_class=SensorStateClass.MEASUREMENT,
         ),
         TuyaSensorEntityDescription(
             key=DPCode.SENSOR_HUMIDITY,
             name="Humidity",
+            icon="mdi:water-percent",
             device_class=SensorDeviceClass.HUMIDITY,
             state_class=SensorStateClass.MEASUREMENT,
         ),
         TuyaSensorEntityDescription(
             key=DPCode.WIRELESS_ELECTRICITY,
             name="Battery",
+            icon="mdi:battery",
             device_class=SensorDeviceClass.BATTERY,
             entity_category=EntityCategory.DIAGNOSTIC,
             state_class=SensorStateClass.MEASUREMENT,
         ),
+        TuyaSensorEntityDescription(
+            key=DPCode.SD_STATUS,
+            name="SD Card Status",
+            icon="mdi:sd",
+            state_class=SensorStateClass.MEASUREMENT,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.SD_FORMAT_STATE,
+            name="SD Card Format State",
+            state_class=SensorStateClass.MEASUREMENT,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        TuyaSensorEntityDescription(
```

Comment:
This DP actually has a fixed unit of measurement, should it be added to the entity description?